src/iuse_actor.cpp

```diff
@@ -3401,7 +3401,7 @@ cata::optional<int> heal_actor::use( Character &p, item &it, bool, const tripoin
 
     // NPCs can use first aid now, but they can't perform firstaid_activity_actor
     // TODO check if NPCs can use the activity actor
-    if( &patient == &p && !p.is_npc() ) {
+    if( !p.is_npc() ) {
```

Comment:
This breaks treating other people because the `firstaid_activity_actor` currently doesn't support that.

https://github.com/CleverRaven/Cataclysm-DDA/blob/a1cea1e8526ed76e86b89e161829299e2b8d7c19/src/activity_actor.cpp#L5733-L5734

Reply:
Right, that's why I didn't remove that. Guess I'll just see about storing the patient so it works on a patient other than yourself?

Reply:
It's probably easiest to do it by character id. Doing it by position like the heal actor does seems kinda error prone.